test/helper/zone-helper.spec.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+describe('zone helper util test', () => {
+  const getNonZoneAwareStack: (err: Error) => string = Error['getNonZoneAwareStack'];
+  const zoneAwareFrames = [
+    'Zone.run', 'Zone.runGuarded', 'Zone.scheduleEventTask', 'Zone.scheduleMicroTask',
+    'Zone.scheduleMacroTask', 'Zone.runTask', 'ZoneDelegate.scheduleTask',
+    'ZoneDelegate.invokeTask', 'ZoneTask.invoke', 'zoneAwareAddListener', 'drainMicroTaskQueue'
+  ];
+
+  const expectStack = function(err: Error) {
+    const simpleStack = getNonZoneAwareStack(err);
+    if (!simpleStack) {
+      return;
+    }
+    const frames = simpleStack.split('\n');
+    for (let i = 0; i < frames.length; i++) {
+      expect(zoneAwareFrames.filter(f => frames[i].indexOf(f) !== -1).length).toBe(0);
```

Comment:
When this fails it will say `was 5 expected 0` which is not useful. 

expect(extraFrames).toEqual([]);

is better since the error would be

`was ['someFrame] expected []`